storage/innobase/handler/handler0alter.cc

```diff
@@ -1467,6 +1467,40 @@ innobase_fts_check_doc_id_col(
 	return(false);
 }
 
+/** Only works for ROW_TYPE=REDUNDANT
+@param[in]	flags	alter table operations flags
+@param[in]	type	table ROW_TYPE
+@return	boolean */
+static bool
+instant_set_nullable_possible(alter_table_operations flags,
+			      const dict_table_t* table)
+{
+	const bool is_redundant = !dict_table_is_comp(table);
+	return flags & ALTER_COLUMN_NULLABLE && is_redundant;
+}
```

Comment:
The `@return` comment is useless, and the `@param type` does not match `table`. I’d dereference `table` only if the `flags` was set.